src/events/pollVotationResults.js

```diff
@@ -10,49 +10,62 @@ module.exports = {
 
     if (
       !author.bot ||
+      !embeds ||
       !Array.isArray(embeds[0]?.fields) ||
       embeds[0]?.data?.type !== 'poll_result'
     ) {
       return;
     }
 
-    const userMentionField = embeds[0]?.fields.find(
-      (field) => field.name === 'poll_question_text'
-    );
+    try {
+      const questionField = embeds[0]?.fields.find(
+        (field) => field.name === 'poll_question_text'
+      );
 
-    if (userMentionField === undefined) {
-      return;
-    }
-
-    const parts = userMentionField.value?.split('|').map((part) => part.trim());
-    if (!Array.isArray(parts) || !parts[1]) {
-      return;
-    }
+      if (!questionField) {
+        return;
+      }
 
-    const userMentioned = `<@${parts[1]}>`;
+      const parts = questionField.value.split('|').map((part) => part.trim());
+      if (!parts || parts.length < 2) {
+        return;
+      }
 
-    const channel = await client.channels.fetch(channelId);
+      const userMentioned = `<@${parts[1]}>`;
+      const isBoostedPoint = parts[0].toLowerCase().includes('boosted');
+      const selectedTagId = isBoostedPoint
+        ? tagIds.boostedPointTagId
+        : tagIds.addPointTagId;
```

Comment:
tagIds.boostedPointTagId doesn't exists after the rebase, it is not on config.js

please use the variables ones are under VOTE_POINTS.TAG_IDS

Reply:
is a constant that is used directly in vote-points

![image](https://github.com/user-attachments/assets/8de08ecb-24cb-415f-966c-d4fb5092661f)

Reply:
AHH thanks for the correction